packages/components/src/internal/util/utils.ts

```diff
@@ -713,3 +714,56 @@ export function getValuesSummary<T>(values: T[], nounSingular: string, nounPlura
     const plural = nounPlural ?? nounSingular + 's';
     return `${values.length} ${plural} (${makeCommaSeparatedString(values)})`;
 }
+
+/**
+ * given a data map, return the CSSProperties that correspond to the 'style' property for that map.
+ * If a column is provided, the map is expected to be a full row of data with the column being (possibly) one of the
+ * fields in that row. If no column is provided, the data is expected to be a single field's data.
+ * @param data either a row of data or a single field's data
+ * @param column the grid column corresponding to the data to extract styling for.
+ */
+export function getDataStyling(data: Map<string, any> | any, column?: GridColumn): CSSProperties {
+    if (!data) {
+        return undefined;
+    }
+    let style;
+    if (column) {
+        style = { textAlign: column.align || 'left' };
```

Comment:
were we just never using this `column.align` property before? It just seems unrelated to the conditional format changes so I was curious to see it here.